app/services/app_integrations/fetch_by.rb

```diff
@@ -0,0 +1,28 @@
+module AppIntegrations
+  class FetchBy
```

Comment:
**question (non-blocking):** Should this be called `FetchOrCreateBy`? It seems like we're creating a new integration if we can't find a target and personally I would not expect a method with the current name to change the record count in our DB.